scripts/tests/test_ft_tracking.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import os
+import tempfile
+import numpy as np
+
+from scilpy import SCILPY_HOME
+from scilpy.io.fetcher import fetch_data, get_testing_files_dict
+
+# If they already exist, this only takes 5 seconds (check md5sum)
+fetch_data(get_testing_files_dict(), keys=['tractograms.zip'])
+tmp_dir = tempfile.TemporaryDirectory()
+
+
+def test_help_option(script_runner):
+    ret = script_runner.run('scil_ft_tracking.py', '--help')
+    assert ret.success
+
+
+def test_execution_(script_runner, monkeypatch):
+    monkeypatch.chdir(os.path.expanduser(tmp_dir.name))
+    in_tractogram = os.path.join(SCILPY_HOME, 'tracking', 'local.trk')
+    in_diameters = os.path.join(SCILPY_HOME, 'tracking', 'diameters.txt')
```

Comment:
I think you should save them in `os.path.expanduser(tmpdir.name)`

Reply:
See Arnaud